### PR TITLE
Add Kingston Communications(KCOM)(AS12390)

### PR DESCRIPTION
**Pull request by thomasdouwes**
Tested with [isbgpsafeyet.com](https://isbgpsafeyet.com)

### CODE REVIEW

File: data/operators.csv [hunk 1]
```diff
@@ -266,6 +266,7 @@ INTERSPACE-MK,cloud,,unsafe,200899,5200
 Monkeybrains,ISP,,unsafe,32329,5215
 BroadbandGibraltarLtd.,ISP,,unsafe,34803,5285
 AltusHost,cloud,,unsafe,51430,5306
+Kingston Communications,ISP,,unsafe,12390,5388
```
Comment (digizeph) [inline data/operators.csv:269]: AS12390 has signed all their prefixes. Suggest change to `partially safe` category.
```suggestion
Kingston Communications,ISP,signed,partially safe,12390,5388
```

**Review by digizeph — APPROVED** [review]

**Comment by thomasdouwes** [comment]
I received an email saying there was a comment on this pull request but I can't find it? what should I do?

**Comment by digizeph** [comment]
@thomasdouwes Thank you for your PR, and sorry for the confusion! we will move forward merging your pull request in the coming days. no action needed from you.

**Comment by thomasdouwes** [comment]
Ok, do I not need to change to signed/partially safe then? I checked on he.net and AS12390 is signed just not filtering. Also, oddly they used to filter but stopped before I made the PR

**Comment by digizeph** [comment]
No need to change your PR. 
In terms of using `partially safe` label, we want to make sure at least some degree of invalid filtering is happening based on our observation. They seem to completely stop filtering RPKI invalid from July 2022. See here: https://stats.labs.apnic.net/rpki/AS12390

File: data/operators.csv (hunk 1 above)
Comment (digizeph) [inline data/operators.csv:269]: You are right about them signing their prefixes. (Sorry about the back-and-forth on the changes, it's on me!)
```suggestion
Kingston Communications,ISP,signed,unsafe,12390,5388
```

**Review by digizeph — CHANGES_REQUESTED** [review]
There is already a AS12390 in the line 240. Should make edits on that line instead.

**Review by digizeph — APPROVED** [review]

**Pull request merged**